### ISSUE TITLE
Notas como strings e não number

### ISSUE THREAD

**Issue by mairaspilack** [reprograma/On3-logica-com-js#1]
Fiz o código abaixo para resolver o exercício de casa, mas as notas coletadas são strings. Como resolvo?

    <script type="text/javascript">
            let nome=prompt("Digite o nome do aluno:");
            let nnota1=prompt("Digite a primeira nota:");
            let nota2=prompt("Digite a segunda nota:");
            let nota3=prompt("Digite a terceira nota:");

            let soma = (nota1 + nota2 + nota3);
            let media = soma/3

            if (media >=7 ) {
            alert(`${nome}, parabéns! você está aprovado(a) com média ${media}, aproveite suas férias!`);
            }
            else {
            confirm (`${nome}, sua média é ${media} e você está reprovado(a).`);
            }
            </script>

**Comment by Asoaresdev** [comment]
Ainda não comecei o exercício, mas lendo sobre javascript, hoje, li algo sobre, e teria que usar uma função a “parseInt( )”, que transforma  a string recebida, no prompt, em número. Esperar a Patricia para esclarecer

**Comment by mairaspilack** [comment]
Pois é, eu pesquisei e li duas coisas:
1. Quando usamos prompt, a informação coletada será sempre interpretada como string (não sei se isso é verdade.
2. Que dá para usar a função parseInt() para converter o número para number, mas nos meus testes eu não consegui, acho que não tô sabendo fazer direito.

**Comment by Asoaresdev** [comment]
Eu usei a parseInt e funcionou aqui!

**Comment by patricia-tonella** [comment]
@mairaspilack @Asoaresdev Sim, podem usar o `parseInt()` para transformar as strings em números.

A sintaxe do parseInt() é bem simples:

```js
parseInt(nome_variavel) 
```

**Comment by mairaspilack** [comment]
Obrigada, @ptonella! Ficou assim:


let nome = prompt("Digite seu nome:");
let nota1 = prompt("Digite a sua nota, de 0 a 10, do primeiro trimestre:");
let nota2 = prompt("Digite a sua nota, de 0 a 10, do segundo trimestre:");
let nota3 = prompt("Digite a sua nota, de 0 a 10, do terceiro trimestre:");

let n1 = parseInt(nota1);
let n2 = parseInt(nota2);
let n3 = parseInt(nota3);

let media = (n1 + n2 + n3)/3
    
if (media >=7 ) {
    alert(`${nome}, parabéns! Você está aprovado(a) com média ${media}, aproveite suas férias!`);
}
else {
    alert (`${nome}, sua média é ${media} e você está reprovado(a).`);
}
